MIGRATION.md

```diff
@@ -537,3 +556,652 @@ import { Grid, GridItem } from '@contentful/f36-components';
+  id="radio"
+  isDisabled={false}
+  isChecked={true}
+  aria-label="some label"
+/>;
+```
+
+### Select
+
+`Select` component becomes compound component in version 4. You can use `Select` and `Select.Option` , with just importing `Select`. You would need to transform your existing code:
+
+```tsx static=true
+<Select id="optionSelect" name="optionSelect" width="large">
+  <Option value="optionOne">Option 1</Option>
+  <Option value="optionTwo">Long Option 2</Option>
+</Select>
+```
+
+into the following:
+
+```tsx static=true
+<Select id="optionSelect" name="optionSelect">
+  <Select.Option value="optionOne">Option 1</Select.Option>
+  <Select.Option value="optionTwo">Long Option 2</Select.Option>
+</Select>
+```
+
+We also introduced some API changes for the select, check the props that had changes:
+
+- `width` was removed;
+- `required` was renamed to `isRequired`;
+- `hasError` was renamed to `isInvalid`;
+- `disabled` was renamed to `isDisabled`;
+- The `Select.Option` component can receive a `isDisabled` prop instead of the `disabled`.
+
+#### How to migrate your Select components
+
+Run the [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod) to migrate your v3 `Select` component to the new version:
+
+`npx @contentful/f36-codemod`
+
+If you decide to do it manually, you would need to transform your code:
+
+```jsx static=true
+import { Select, Option } from '@contentful/forma-36-react-components';
+
+<Select name="optionSelect" id="optionSelect">
+  <Option value="optionOne">Option One</Option>
+  <Option value="optionTwo">Option Two</Option>
+  <Option value="optionThree" disabled>
+    Option Three
+  </Option>
+</Select>;
+```
+
+into this new version:
+
+```jsx static=true
+import { Select } from '@contentful/f36-components';
+
+<Select name="optionSelect" id="optionSelect">
+  <Select.Option value="optionOne">Option One</Select.Option>
+  <Select.Option value="optionTwo">Option Two</Select.Option>
+  <Select.Option value="optionThree" isDisabled>
+    Option Three
+  </Select.Option>
+</Select>;
+```
+
+### Switch
+
+In v4 the `Switch` component has received API improvements, that are based on our code style guide, which creates consistent, easy-to-use APIs. For example:
+
+```jsx static=true
+import { Switch } from '@contentful/forma-36-react-components';
+
+<Switch name="switch" id="switch" onToggle={onToggle} labelText="some label" />;
+```
+
+will become:
+
+```jsx static=true
+import { Switch } from '@contentful/f36-components';
+
+<Switch name="switch" id="switch" onChange={onChange}>
+  some label
+</Switch>;
+```
+
+When changing `onToggle` to `onChange`, before it would expect a function that receives boolean value, now it works as any other `onChange` on any form component, so the handling of on or off should be done by the user.
+
+#### How to migrate your Switch components
+
+The migration for the Switch needs to be done manually by updating the usage of the props, for example:
```

Comment:
You must manually migrate the Switch component by updating the props usage. For example: